.travis.yml

```diff
@@ -85,3 +85,4 @@ before_cache:
   # Delete python bytecode to prevent cache rebuild.
   - find third_party -name "*.pyc" -print -delete
   - find backend_prod_files/extensions/ -name "*.pyc" -print -delete
+  - rm -r node_modules/.cache/cache-loader
```

Comment:
should we remove the entire ./.cache/ folder? (Considering we don't know what else get cached here or what else can be cached in future.)

Reply:
At least on my machine `cache-loader` is the only thing in `.cache`

Reply:
I also have ./.cache/terser-webpack-plugin/*

Reply:
Interesting. Do the files in `terser-webpack-plugin/*` contain Oppia code or other code that shouldn't be cached?

Reply:
I'm unable to find the .cache folder in my repo now, but IIRC there wasn't any oppia specific code inside the `terser-webpack-plugin` dir.